explainer_worker/worker_sql_db.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import select, update
+
+from tabels import Upload, _session, Status
+
+
+class sqldb:
+    def get_uploads_by_status(self, status):
+        query = select(Upload).where(Upload.status == status)
+        with _session() as s:
+            return s.execute(query).scalars().all()
+
+    def update_uploadds_by_id_list(self, id_list, updates_dict):
+        query = update(Upload).where(Upload.id.in_(id_list)).values(**updates_dict)
+        with _session() as s:
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names
Try to avoid i and j as well if there is a better name available.